lib/assets/bundle.jst

```diff
@@ -40,4 +40,4 @@ var ${ exportName };
         ${ exportName } = buildBemXjst(${ globalDependencies });
         global['${ exportName }'] = ${ exportName };
     }
-})(typeof window !== "undefined" ? window : global);
+})(typeof window !== "undefined" ? window : global ? global : this);
```

Comment:
@sbmaxx Sorry, `global` works fine in perl:v8? `global ? global : this === global || this`